great_expectations/expectations/self_check_util.py

```diff
@@ -1228,7 +1228,11 @@ def check_json_test_result(test, result, data_asset=None):
                 elif isinstance(data_asset, (SqlAlchemyBatchData, SparkDataFrame)):
                     pass
                 else:
-                    assert result["result"]["unexpected_index_list"] == value
+                    assert result["result"]["unexpected_index_list"] == value, (
+                    "unexpected_index_list expected "
+                    + str(value)
+                    + " but got "
+                    + str(result["result"]["unexpected_index_list"]))
```

Comment:
Probably a linter issue here
```suggestion
                    + str(result["result"]["unexpected_index_list"])
                )
```